api/graphql/types/__init__.py

```diff
@@ -0,0 +1,207 @@
+import strawberry
```

Comment:
I think for the most part these types would be better placed being alongside the mutations they are used in